tests/cephfs/cephfs_utilsV1.py

```diff
@@ -5194,3 +5197,85 @@ def get_client_id(self, client, fs_name, mounted_dir):
         client_id = client_id.replace('"', "").replace("\n", "")
         log.info(f"Client ID : {client_id} for Mounted Directory : {mounted_dir}")
         return client_id
+
+    def set_subvolume_earmark(
+        self, client, vol_name, subvol_name, earmark, group_name=None
```

Comment:
Can we have keyword arguments(**kwargs) similar to other libraries that can help in future enhancement of the function? Not a mandate stuff, but you can think through

Reply:
yeah. that works well. as well. but i thought that this way is more intuitive.